src/components/Layouts/ReportLayout.js

```diff
@@ -68,8 +68,10 @@ class ReportLayout extends Component {
       // set dynamic height for all sections, fix top attribute.
       const itemsByRow = groupBy(Object.values(this.itemElements), item => item.gridItem.y);
       const heightMap = {};
-      Object.keys(itemsByRow).sort(compareFields).forEach((key) => {
-        const items = itemsByRow[key];
+      const rows = Object.keys(itemsByRow);
+      Object.keys(itemsByRow).sort(compareFields).forEach((key, index) => {
+        const items = itemsByRow[rows[index]];
+        const nextRowHeight = this.getMaxHeight(itemsByRow[rows[index + 1]]);
```

Comment:
shouldnt you check `rows[index + 1]` exists?

Reply:
no need, 
getMaxHeight handles it